### PR TITLE
Release v0.9.0

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Cut a new release to be used in E2E testing

#### Ticket Link
https://community.mattermost.com/core/pl/9rbacrfxeibxud35r3keh3k5me

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -20,91 +20,119 @@
...
-            "default": "demo_plugin"
-        }, {
-            "key": "secretNumber",
-            "display_name": "Secret Number",
-            "type": "number",
-            "help_text": "A secret number that the demo plugin will watch for. If the secret number is mentioned in any channel then the demo plugin will publish a special message.",
-            "placeholder": "Some secret number",
-            "default": 123
-        }]
+        "settings": [
```
Comment (hanzei) [inline plugin.json:23]: I also fixed some indentation issues within in this PR

**Review by jfrerich — COMMENTED** [review]
Yes, please! Thanks, @hanzei!

**Comment by hanzei** [comment]
@jfrerich Did you meant to approve this PR?

**Review by jfrerich — APPROVED** [review]
@hanzei, yes I meant to approve this, but only commented.  Approving now!  Thanks!

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Ran through release testing
- TM4J tests are in place for newly added functionality
- No issues found

LGTM!

**Pull request merged**